test/pricer_rule/utils.js

```diff
@@ -60,24 +62,30 @@ module.exports.createOrganization = async (accountProvider) => {
  * Creates and returns the tuple:
  *      (tokenRules, organizationAddress, token)
  */
-module.exports.createTokenEconomy = async (accountProvider) => {
+module.exports.createTokenEconomy = async (accountProvider, config = {}, eip20TokenConfig = {}) => {
```

Comment:
Please, let's document `config` and `eip20TokenConfig`, all fields, and how do they change the function behavior.